src/main/java/pl/javastart/task/model/TicketCash.java

```diff
@@ -0,0 +1,86 @@
+package pl.javastart.task.model;
+
+import java.util.Arrays;
+
+public class TicketCash {
+    private static final int INITIAL_EVENT_MAX = 3;
+    private static final double STUDENT_DISCOUNT = 0.23;
+    private static final double PASSPORT_DISCOUNT = 0.5;
+    private static final double INVALID_VALUE = -1;
+    private int currentTicketId = 1;
```

Comment:
To też jakieś rozwiązanie. Gdzieś w zamyśle tego zadania było używanie statycznego pola, ale tak jak zrobiłeś też jest poprawnie.